mockserver/update_state.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
do we still need those?

Reply:
They prevent Python 2 contributors from putting syntax error into Python 3 code.  They also enable the __end__ parameter in Python 2 and prohibit print statements with trailing comma.  If you are 100% certain that no one will try any of these things or paste in code that tries any of these things, then we can take the future imports out.

Reply:
If we switch completely to Python3 then this should not be necessary, so we can go ahead and remove them!